allenact/algorithms/onpolicy_sync/runner.py

```diff
@@ -86,14 +89,18 @@ def __init__(
         disable_tensorboard: bool = False,
         disable_config_saving: bool = False,
         distributed_ip_and_port: str = "127.0.0.1:0",
+        distributed_preemption_threshold: float = 0.7,
```

Comment:
This is nice - not sure why it was never added as an argument, but good you did. I guess it would also make sense to make it an arg in main?

Reply:
Sorry, have just been periodically adding to this branch for any issues I’ve come across. I think I’m just about ready for it, other than addressing the existing comments above. I’ve been waiting a bit since it’s nice to sometimes add arguments to the callbacks, but if it’s merged into main, it’d be hard to add anything without breaking backwards compatibility.